jenkins-scripts/dsl/_configs_/Globals.groovy

```diff
@@ -11,7 +11,7 @@ class Globals
    static gazebodistro_branch = false
 
    static CRON_EVERY_THREE_DAYS = 'H H * * H/3'
-   static CRON_HOURLY = 'H H * * *'
+   static CRON_HOURLY = 'H * * * *'
```

Comment:
This string will run at an arbitrary minute of the hour `H`, of any hour of the day, any day of the month, any month of the year, any day of the week.